setup.py

```diff
@@ -84,6 +85,7 @@ def get_extra_requires() -> dict:
             "fastapi",
             "pettingzoo[mpe]",
             "pettingzoo[butterfly]",
+            "envpool",
```

Comment:
delete this. Add a README.md in examples/envpool. And show how to install dependencies in the markdown.